dashboard/app/templates.html

```diff
@@ -126,7 +126,12 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 	{{range $b := .Bugs}}
 		<tr>
 			{{if $.ShowNamespace}}<td>{{$b.Namespace}}</td>{{end}}
-			<td class="title"><a href="{{$b.Link}}">{{$b.Title}}</a></td>
+			<td class="title">
+				{{if gt $b.PatchedOn 0}}
+				<span class="attention">[patched]</span>
```

Comment:
What was the confusion?

Reply:
Doing Ctrl+F on the main web page and not noticing that the bug is already fixed.

Reply:
Then it's better to do with Javascript.

But looking at this and "dashobard: highlight patched bugs" patch, I am thinking perhaps we should remove "fix pending" bugs from the main page entirely and move to "fixed" page. It would remove the need in both of these changes, right?
It was useful initially and before syzbot started sending notifications about wrong fixing patches. But I am not sure anybody is looking closely at it (besides total number of "fix pending" bugs to detect unhealthy situations, but that's useful only to people doing overall monitoring so belong elsewhere).

Reply:
> I am thinking perhaps we should remove "fix pending" bugs from the main page entirely

That sounds interesting. Would you then prefer to see there two separate tables (with pending fixes and then with completed) or just merge it into one table and include the "Patched" column?

Reply:
Code-wise it will be easier if it's just one table

Reply:
> That sounds interesting. Would you then prefer to see there two separate tables (with pending fixes and then with completed) or just merge it into one table and include the "Patched" column?

I don't have strong preference. It will be used mostly by us.
But if we show it in one table I think it will be reasonable to sort by "not patched on all instances" flag and show them at the top.

Reply:
Pushed the updated changes (not tested yet).